recipes/vlfeat/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.9.20" %}
+
+package:
+  name: vlfeat
+  version: {{ version }}
+
+source:
+  fn: vlfeat-{{ version }}-bin.tar.gz
+  url: http://www.vlfeat.org/download/vlfeat-{{ version }}-bin.tar.gz
+  
+  patches:
+    - no_snprintf.patch  # [win and py35]
+    - no_crt_copy.patch  # [win]
+
+build:
+  features:
+    - vc9   # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}                # [win]
```

Comment:
Yes, that sounds like a bug to me.  Please file an issue first.  If you have time to work on it, I'd certainly appreciate it, but don't feel obligated.